example/index.html

```diff
@@ -25,7 +25,10 @@ <h1>GitHub for Dart - Demos</h1>
     querystring parameter to specify your token on any of the examples like
     <b><code>?token=[yourtoken]</code></b>
 
-    <p>Github Token: <input type="text" id="token" size="50"></p>
+    <p>
+      Github Token: <input type="text" id="token" size="50">
+      <button id=removeTokenBtn>Clear Github Token</button>
+    </p>
```

Comment:
Switched to localStorage to preserve your token longer and provide a clear button to remove it.